boards/aithinker/ai_wb2_12f/doc/index.rst

```diff
@@ -0,0 +1,108 @@
+.. zephyr:board:: ai_wb2_12f
+
+Overview
+********
+
+BL602/BL604 is a Wi-Fi+BLE chipset introduced by Bouffalo Lab, which is used
+for low power consumption and high performance application development.  The
+wireless subsystem includes 2.4G radio, Wi-Fi 802.11b/g/n and BLE 5.0
+baseband/MAC design.  The microcontroller subsystem includes a 32-bit RISC CPU
+with low power consumption, cache and memory.  The power management unit
+controls the low power consumption mode.  In addition, it also supports
+various security features.  The external interfaces include SDIO, SPI, UART,
+I2C, IR remote, PWM, ADC, DAC, PIR and GPIO.
+
+The BL602 Development Board features a SiFive E24 32 bit RISC-V CPU with FPU,
```

Comment:
E.g this

Reply:
I don't mind the copy-paste of the blurb but just fix the name of the board :)

Reply:
do you mean 'The BL602 Development Board' ?
This BL602 Development Board? The BL602 WB2 12F Module Development Board?